packages/logger/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@apppium/logger",
+  "version": "1.0.0",
+  "author": "https://github.com/appium",
+  "description": "A Universal Logger For The Appium Ecosystem",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/appium/appium.git",
+    "directory": "packages/logger"
+  },
+  "main": "build/index.js",
+  "types": "index.ts",
+  "files": [
+    "index.ts",
+    "lib",
+    "build",
+    "tsconfig.json",
+    "!build/tsconfig.tsbuildinfo",
+    "CHANGELOG.md"
+  ],
+  "directories": {
+    "lib": "./lib"
+  },
+  "scripts": {
+    "test": "npm run test:unit",
+    "test:smoke": "node ./build/index.js",
+    "test:unit": "mocha --exit --timeout 1m \"./test/unit/**/*-specs.js\""
+  },
+  "dependencies": {
+    "console-control-strings": "1.1.0",
+    "set-blocking": "2.0.0"
+  },
+  "license": "Apache-2.0",
```

Comment:
@lukekarrys Would you kindly approve changing of the original ISC license to Apache-2.0 for this npmlog fork?

Reply:
@KazuCocoa Please do not forget to update this field as well if we keep the ISC license